### PR TITLE
Several driver specific api enhancements

### PR DESCRIPTION
**Pull request by jwkoelewijn**
- create-session: Changed this into a multimethod dispatching on driver
type. The Geckodriver now returns the sessionId in map under the :value
key, rather than under the top-level :sessionId key
- click: Wait a bit when using the :safari driver
- fill-el: The geckodriver wants the text to be available under a :text
key in the params, rather than under the :value key. It also expects a
string instead of a char sequence.

### CODE REVIEW

**Comment by igrishaev** [comment]
@jwkoelewijn thank you for your efforts, tour changes look sensible. But before we merger them, we need to implement some kind of driver-version dispatchering mechanism. It seems you use the laters version of drivers whereas other people use legacy ones. Such update will break their pipeline. I created an issue #59 to consider not only browser type but also the driver version. Once I add such functionality, we will ship your changes.

**Comment by jwkoelewijn** [comment]
Thanks for the update, makes sense, completely didn't consider older versions... :S

**Comment by igrishaev** [comment]
@jwkoelewijn btw you please submit your os and drivers version?

**Comment by jwkoelewijn** [comment]
Yes, no problem:

I'm running MacOS High Sierra 10.13.1 Beta (17B25c).
geckodriver version: 0.19.1
chromedriver version: 2.33.506106 (8a06c39c4582fbfbab6966dbb1c38a9173bfb1a2)

It seems to be impossible to get a version for the safaridriver, but I'm guessing it comes with Safari itself, which has version 11.0.1 (13604.3.1)

Hope this answers your question.

File: src/etaoin/api.clj [hunk 1]
```diff
@@ -140,25 +140,38 @@
...
 
-(defn create-session
+(defmulti create-session
   "Initiates a new session for a driver. Opens a browser window as a
   side-effect. All the further requests are made within specific
   session. Some drivers may work with only one active session. Returns
   a long string identifier."
+  dispatch-driver)
+
+(defmethod create-session :default
```
Comment (igrishaev) [inline src/etaoin/api.clj]: @jwkoelewijn this one was fixed in the latest `0.1.9` release.

File: src/etaoin/api.clj [hunk 2]
```diff
@@ -710,11 +723,21 @@
     [:session (:session @driver) :element el :click]
     nil _))
 
-(defn click
+(declare wait)
+
+(defmulti click
```
Comment (igrishaev) [inline src/etaoin/api.clj]: @jwkoelewijn in 0.2.0-snapshot, I implemented `click` as a multimethod, so you are welcome to extend it locally. I believe that 1 second wait for Safari requirement depends on your local env, OS, etc.

File: src/etaoin/api.clj [hunk 3]
```diff
@@ -1828,12 +1851,26 @@
   [driver text & more]
   (apply fill-active* driver text more))
 
-(defn fill-el
```
Comment (igrishaev) [inline src/etaoin/api.clj]: @jwkoelewijn that issues with filling  elements was fixed in some prev. version. Check the latest master please.

**Comment by igrishaev** [comment]
@jwkoelewijn I grabbed most of your changes. Thank you for your suggestions. I added you into contributors list. Will check your next PR soon.

**Pull request closed without merging**